### PR TITLE
Characteristic Function True/False to 1/0 fix

### PR DESCRIPTION
**Pull request by havijw**
I added documentation to `SetVal.__call__` and changed the indicator functions to return `1`/`0` instead of `True`/`False`. I also made a notebook with some tests; I'm not sure what the best place for that kind of thing is so that file can be deleted before merging into the main branch.

It looks like there's actually a way to attach files to a pull request so maybe that's the way I can send tests in the future.

### CODE REVIEW

**Review by EasyArray — CHANGES_REQUESTED** [review]
Testing the review process: 

Maybe rephrase "indicator function" as "characteristic function, true of the set's members" ?

**Review by EasyArray — CHANGES_REQUESTED** [review]
Oh, and do maybe remove the test file? Not sure what the best procedure is....

**Comment by havijw** [comment]
> Testing the review process:
> 
> Maybe rephrase "indicator function" as "characteristic function, true of the set's members" ?

Made this change — not sure why I had "indicator function" in my head!

> Oh, and do maybe remove the test file? Not sure what the best procedure is....

I just deleted it in the most recent version of this branch. I think in the future I'll attach any test files I had to the pull request since they don't need to be part of the repository but could ostensibly be useful to have.

I think it should be ready to merge now! In general I'll try to rebase whatever branch I'm working on frequently enough that there aren't any conflicts when merging with main and the history can stay linear.
